app/cmd/runner.go

```diff
@@ -0,0 +1,133 @@
+		i18n.T("If you want to download Jenkins as LTS"))
+}
+
+var runnerCmd = &cobra.Command{
+	Use:   "runner",
+	Short: i18n.T("The wrapper of jenkinsfile runner"),
+	Long: i18n.T(`The wrapper of jenkinsfile runner
+Get more about jenkinsfile runner from https://github.com/jenkinsci/jenkinsfile-runner`),
+	RunE: func(cmd *cobra.Command, _ []string) (err error) {
+
+		var userHome string
+		if userHome, err = homedir.Dir(); err != nil {
+			return err
+		}
+		//Start by downloading the mirror for the jenkinsfileRunner
+		jenkinsfileRunnerVersion := fmt.Sprintf("jenkinsfile-runner-%s.jar", runnerOption.JfrVersion)
+		jenkinsfileRunnerTargetPath := fmt.Sprintf("%s/.jenkins-cli/cache/%s/%s", userHome, runnerOption.WarVersion, jenkinsfileRunnerVersion)
+		jenkinsfileRunnerURL := fmt.Sprintf("https://repo.jenkins-ci.org/list/releases/io/jenkins/jenkinsfile-runner/jenkinsfile-runner/1.0-beta-11/%s", jenkinsfileRunnerVersion)
+		logger.Info("Prepare to start Downloading jenkinfileRunner", zap.String("URL", jenkinsfileRunnerURL))
+		downloader := util.HTTPDownloader{
+			URL:            jenkinsfileRunnerURL,
+			ShowProgress:   true,
+			TargetFilePath: jenkinsfileRunnerTargetPath,
+		}
+		if err := downloader.DownloadFile(); err != nil {
+			//Fatal error has occured while downloading the file.
+			log.Fatal(err)
+		}
+
+		if runnerOption.WarPath == "" {
+			// If it does not exist download the jenkins.war
+			jenkinsWar := fmt.Sprintf("%s/.jenkins-cli/cache/%s/jenkins.war", userHome, runnerOption.WarVersion)
+			logger.Info("prepare to download jenkins.war as pre-requisite for jfr", zap.String("localPath", jenkinsWar))
+
+			if _, fileErr := os.Stat(jenkinsWar); fileErr != nil {
+				download := &CenterDownloadOption{
+					Mirror:       "default",
+					Output:       jenkinsWar,
+					ShowProgress: true,
+					LTS:          runnerOption.LTS,
+					Version:      runnerOption.WarVersion,
+				}
+				if err = download.DownloadJenkins(); err != nil {
+					return err
+				}
+			}
+		}
+
+		//Check if jenkins war path exists
+		if runnerOption.WarPath != "" && filepath.Ext(strings.TrimSpace(runnerOption.WarPath)) != ".war" {
+			return fmt.Errorf("incorrect file path : %s", runnerOption.WarPath)
+		}
+
+		//Check if plugin.txt path is provided
+		if runnerOption.PluginPath == "" {
+			return fmt.Errorf("plugin.txt path is not provided kinldy proivde path")
+		}
+
+		//Check if the plugin file has a valid extension
+		if filepath.Ext(strings.TrimSpace(runnerOption.PluginPath)) != ".txt" {
+			return fmt.Errorf("incorrect file type it should be a txt file : %s", runnerOption.PluginPath)
+		}
+
+		//Check if jenkinsfile path is empty
+		if runnerOption.PluginPath == "" {
+			return fmt.Errorf("kindly provide valid path to jenkinsfile")
+		}
+
+		if filepath.Base(runnerOption.JenkinsfilePath) != "Jenkinsfile" {
+			return fmt.Errorf("invalid file type. kindly provide a jenkinsfile")
+		}
+
+		fmt.Println("We are Definitely reaching here")
```

Comment:
Please use cmd.Println instead of fmt.Println.

Because of it can support output to a different place.

Reply:
Yes :+1:

Reply:
Still you need to use cmd.Println instead of fmt.Println